oauth2_provider/views/introspect.py

```diff
@@ -24,7 +24,9 @@ class IntrospectTokenView(ClientProtectedScopedResourceView):
     @staticmethod
     def get_token_response(token_value=None):
         try:
-            token = get_access_token_model().objects.get(token=token_value)
+            token = get_access_token_model().objects.select_related(
```

Comment:
code updates need unit tests in general, can you re check?

Reply:
This can be verified in tests with [`assertNumQueries`](https://docs.djangoproject.com/en/dev/topics/testing/tools/#django.test.TransactionTestCase.assertNumQueries).

Reply:
@kevin-brown thanks for telling :)